cfpb/capital-framework
# Add `u-svg-inline-bg` mixin for inline SVG background

## Additions
- Adds `.u-svg-inline-bg( @name )` mixin and associated plugin.

## Removals
- Remove cf-core dependency from cf-icons.
- Removes `@cf-icon-path` variable.

## Changes
- Convert cf-tables to use SVG image instead of PNG for sort button.
- Adds missing "font variables" entry from cf-typography TOC.

## Testing

1. `gulp build && gulp docs && bundle exec jekyll serve` should pass and launch the docs.
2. Visit http://localhost:4000/components/cf-tables/#sortable-tables and compare to https://cfpb.github.io/capital-framework/components/cf-tables/#sortable-tables
3. Visit http://localhost:4000/components/cf-forms/#select-dropdown and compare to https://cfpb.github.io/capital-framework/components/cf-forms/#select-dropdown
4. Visit http://localhost:4000/components/cf-forms/#checkboxes-and-radio-inputs and compare to https://cfpb.github.io/capital-framework/components/cf-forms/#checkboxes-and-radio-inputs

## Screenshots

Before:
<img width="422" alt="Screen Shot 2019-07-09 at 11 22 34 AM" src="https://user-images.githubusercontent.com/704760/60902171-322c9c80-a23d-11e9-80ee-f9f0b84600df.png">

After:
<img width="438" alt="Screen Shot 2019-07-09 at 11 32 08 AM" src="https://user-images.githubusercontent.com/704760/60902188-38bb1400-a23d-11e9-82f3-a9b73640024b.png">


## Notes

- We may want to scrutinize whether packages other than cf-icons also do not depend on cf-core and remove those dependencies.
